setup.cfg

```diff
@@ -7,11 +7,16 @@ per-file-ignores =
   vitessce/__init__.py: F401
   vitessce/data_utils/__init__.py: F401
 ignore =
-  E501, # Ignore line too long
-  W605, # Ignore invalid escape sequence '\*'
-  W503, # Ignore line break before binary operator: Skim down the left edge to understand intent.
-  E127 # Ignore continuation line over-indented for visual indent
-  E128 # Ignore continuation line under-indented for visual indent
+  # Ignore line too long
+  E501,
+  # Ignore invalid escape sequence '\*'
+  W605,
+  # Ignore line break before binary operator: Skim down the left edge to understand intent.
+  W503,
+  # Ignore continuation line over-indented for visual indent
+  E127
+  # Ignore continuation line under-indented for visual indent
+  E128
```

Comment:
fixing https://stackoverflow.com/questions/74558565/flake8-error-code-supplied-to-ignore-option-does-not-match-a-z1-30